### PR TITLE
update tabulator-tables add TooltipModule

### PR DESCRIPTION
**Pull request by maku**
Add TooltipModule

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## tabulator-tables ([<kbd>unpkg</kbd>](https://unpkg.com/browse/tabulator-tables@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'tabulator-tables'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.

The most common way to resolve this error is to use 'export =' syntax.
To learn more about 'export =' syntax, see https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 97bf7c281289e9eaceaa154d9c4114be46aab31b
</p>

**Review by mike-lischke — CHANGES_REQUESTED** [review]
Please add a simple test for this change. Can be as easy as registering the module in the test (like it is done for most of the other modules).

File: types/tabulator-tables/tabulator-tables-tests.ts [hunk 1]
```diff
@@ -1039,3 +1039,5 @@ table = new Tabulator('#test', {
 });
 table.on('dataSorting', ([sorter]) => sorter.field);
 table.on('dataSorted', ([sorter]) => sorter.field);
+
+Tabulator.registerModule([TooltipModule]);
```
Comment (mike-lischke) [inline types/tabulator-tables/tabulator-tables-tests.ts:1043]: Just a few lines up there's already a `registerModule` call, but well...

**Review by mike-lischke — APPROVED** [review]

**Comment by maku** [comment]
Ready to merge

**Pull request merged**